dashboard/src/components/Header/Header.tsx

```diff
@@ -117,7 +117,7 @@ class Header extends React.Component<IHeaderProps, IHeaderState> {
                 <ul className="header__nav__menu" role="menubar">
                   {this.links.map(link => (
                     <li key={link.to}>
-                      <HeaderLink {...link} currentNamespace={namespace.current} />
+                      <HeaderLink {...link} currentCluster={namespace.cluster} currentNamespace={namespace.current} />
```

Comment:
we probably want to store the `cluster` in a different object. `namespace.cluster` is a bit confusing.

Reply:
Cool, thanks for the explanation. I'd go with the second too so we can populate the selectors and show everything. I am not worried about refreshing this data since it's just a matter of refreshing the page.

Reply:
Great, I started that yesterday, should have it done today.